src/components/link/Link.stories.js

```diff
@@ -0,0 +1,63 @@
+import { html } from 'lit';
+import './link';
+import arrowRightIcon from '@carbon/icons/es/chevron--right/16';
+
+export default {
+  title: 'Components/Link',
+  component: 'kd-link',
+  argTypes: {
+    kind: {
+      control: { type: 'select' },
+      options: ['primary', 'secondary'],
+    },
+    target: {
+      control: { type: 'select' },
+      options: ['_self', '_blank', '_top', '_parent'],
+    },
+  },
+};
+
+const args = {
+  href: '',
+  target: '_self',
+  kind: 'primary',
+  disabled: false,
+};
+
+const linkIconArgs = { ...args, sizeOverride: 16 };
+
+// Inline link
+export const Inline = {
+  args: args,
+  render: (args) =>
+    html`
+      <kd-link
+        inline
```

Comment:
Wire up this arg and fix the boolean to default to false.

Reply:
Inline links are default for link so we need to make **inline** true default. Hence I put condition to apply classes at below function. 

   `private returnClassMap() {

    if (this.disabled) {
      return classMap({
        ['kd-link-text-disabled']: this.disabled,
      });
    } else {
      return classMap({
        ['kd-link-text-primary']:
          this.kind === LINK_TYPES.PRIMARY || !this.kind,
        ['kd-link-text-secondary']: this.kind === LINK_TYPES.SECONDARY,
        ['kd-link-text-inline']: this.inline,
        ['kd-link-text-standalone']: !this.inline,
      });
    }}`

Reply:
Can you help me here?

Reply:
It's just that any Boolean properties you create on components have to default to false or they won't work properly. So instead of prop `inline = true`, change to `standalone = false`

Reply:
ok. Instead of `inline: true;` I put `standalone = false;` bydefault.